scripts/uitest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# Define paths to uitest directories and command.
+script_dir=`dirname $0`
+root_dir=`realpath $script_dir/../`
+uitest_dir=$root_dir/uitest
+uitest=$uitest_dir/devenv/bin/uitest
+
+# Define default parameters.
+cloud="lxd"
+archive=`ls $root_dir/dist/jujugui-*.tar.bz2`
```

Comment:
Should we have a check here for multiple returned values? I've frequently had multiple versions in the dist folder and it'll likely become more likely as we speed up the release process.